_CodingChallenges/134-heart-curve.md

```diff
@@ -17,6 +17,19 @@ videos:
     url: "/Tutorials/9-additional-topics/9.20-polar-coordinates"
   - title: "Drawing with Fourier Transform and Epicycles"
     url: "/CodingChallenges/130.1-fourier-transform-drawing"
+    
+    
```

Comment:
remove this line

Reply:
i think you also need to remove last two lines

Reply:
yeah, this is frustrating and confusing. but I said last two lines, not all the last lines (the rules are very strict)

**you can refer to other peoples contribution to see which is the correct syntax**